### PR TITLE
Network February release

### PR DESCRIPTION
**Pull request by anton-evseev**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>

Please merged https://github.com/Azure/azure-rest-api-specs/pull/5223 first

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by number213** [comment]
"SDK Generation - Python" generation keeps failing with OOM error

**Comment by dsgouda** [comment]
startbuild

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by dsgouda** [comment]
startbuild

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by number213** [comment]
@dsgouda I keep re-running CI job but it keeps failing with Out of Memory exception. Could you please investigate or merge (all of those changes were reviewed when they were merged into network-february-release)?

File: specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+            "BeginsWith",
+            "EndsWith",
+            "Regex"
+          ],
+          "x-ms-enum": {
+            "name": "WebApplicationFirewallOperator",
+            "modelAsString": true
+          }
+        },
+        "negationConditon": {
```
Comment (nschonni) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json]: Conditon -> Condition

Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json]: Server-side typoe, nothing we can do
https://github.com/Azure/azure-rest-api-specs/pull/5341#discussion_r263960243

Comment (nschonni) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json]: I guess the next API version would fix it then?

Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json]: That's a question for @azhao155

**Comment by dsgouda** [comment]
@number213 We are working on it.

**Comment by dsgouda** [comment]
startbuild

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by dsgouda** [comment]
@number213 You can ignore the Net Sdk automation error for now, shouldn't block rest of the review.
We will let you know what would be the best course of action for Sdk generation

**Comment by dsgouda** [comment]
startbuild

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by dsgouda** [comment]
@jianghaolu Feel free to merge when ready

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by dsgouda** [comment]
@jianghaolu @number213 Please ignore the Net Sdk generation failure and merge this PR if ready

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Pull request merged**